3.processing_features/plate_info_dictionary.yaml

```diff
@@ -4,9 +4,9 @@ Plate_3:
 Plate_3_prime:
   dest_path: data/converted_data/Plate_3_prime.parquet
   source_path: /home/jenna/nf1_cellpainting_data/2.cellprofiler_analysis/analysis_output/Plate_3_prime/Plate_3_prime_nf1_analysis.sqlite
-Plate_4:
-  dest_path: data/converted_data/Plate_4.parquet
-  source_path: /home/jenna/nf1_cellpainting_data/2.cellprofiler_analysis/analysis_output/Plate_4/Plate_4_nf1_analysis.sqlite
-Plate_5:
-  dest_path: data/converted_data/Plate_5.parquet
-  source_path: /home/jenna/nf1_cellpainting_data/2.cellprofiler_analysis/analysis_output/Plate_5/Plate_5_nf1_analysis.sqlite
+# Plate_4:
```

Comment:
Why are these commented?

Reply:
Ahh its because I use this run specific plates for preprocessing. I will fix this.